packages/astro/src/core/errors/errors-data.ts

```diff
@@ -395,8 +395,11 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 	},
 	/**
 	 * @docs
+	 * @message
+	 * **Example error messages:**<br/>
+	 * InvalidPrerenderExport: A `prerender` export has been detected, but its value cannot be statically analyzed.
 	 * @description
-	 * A `prerender` export was detected, but the value was not statically analyzable. Values computed at runtime are not supported, so `export const prerender` can only be set to `true` or `false`. Variables are not supported.
+	 * The `prerender` feature only supports a subset of valid JavaScript—be sure to use exactly `export const prerender = true` so that our compiler can detect this directive at build time. Variables, `let`, and `var` declarations are not supported.
```

Comment:
```suggestion
	 * The `prerender` feature only supports a subset of valid JavaScript — be sure to use exactly `export const prerender = true` so that our compiler can detect this directive at build time. Variables, `let`, and `var` declarations are not supported.
```
I suspect the spaces will help for readability. (These could also be two separate sentences.) But otherwise, I like the explicit "use exactly this" message!